src/cortxfs/cortxfs_ops.c

```diff
@@ -762,94 +769,126 @@ int cfs_rename(struct cfs_fs *cfs_fs, cfs_cred_t *cred,
 		/* Remove the actual 'destination' object only if all
 		 * previous operations have completed successfully.
 		 */
-		log_trace("Removing detached file (%llu)", dino);
-		RC_WRAP_LABEL(rc, out, cfs_destroy_orphaned_file, cfs_fs,
-			      &dino);
+		log_trace("Removing detached file (%llu)", *dst_ino);
```

Comment:
Is it possible that dst_ino could be  NULL here and you are trying to reference the value at that NULL pointer? If yes, please check line 794 on how to handle this.

Reply:
It won't be null as it is inside `overwrite_dst` and `dst_ino` is derived if we are overwriting the destination. So not needed a NULL check here.